platform/ios/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Style layers
 
 * Added support for aggregate expressions as input values to `MGL_MATCH` expressions. ([#11866](https://github.com/mapbox/mapbox-gl-native/pull/11866))
+* Fixed a crash caused by `literal` expressions when used in non-data-driven properties. ([#11960](https://github.com/mapbox/mapbox-gl-native/pull/11960))
```

Comment:
This fix made it into `ios-v4.0.2` (via #11961), so we don’t need to changelog it here.